x-pack/test/alerting_api_integration/spaces_only/tests/alerting/monitoring.ts

```diff
@@ -96,6 +96,32 @@ export default function monitoringAlertTests({ getService }: FtrProviderContext)
       expect(getResponse.body.monitoring.execution.history[4].success).to.be(false);
       expect(getResponse.body.monitoring.execution.calculated_metrics.success_ratio).to.be(0.6);
     });
+
+    it('should populate rule objects with the calculated percentiles', async () => {
```

Comment:
Should we add a test to verify percentiles are being calculated properly? Maybe like verify they are there and verify the `p99` is equal to the highest duration, or something?

Reply:
I feel like that's testing the implementation of the library, although i still did it in the unit test for the monitoring util

Reply:
Okay sounds good!